lib/orbf/rules_engine/fetch_data/orgunit_facts.rb

```diff
@@ -0,0 +1,37 @@
+
+module Orbf
+  module RulesEngine
+    class OrgunitFacts
+      def initialize(org_unit, pyramid)
+        @org_unit = org_unit
+        @pyramid = pyramid
+      end
+
+      def to_facts
+        level_facts.merge(to_group_set_facts)
+      end
+
+      private
+
+      attr_reader :org_unit, :pyramid
+
+      def level_facts
+        parent_ids = org_unit.path.split("/").reject(&:empty?)
+        level_facts = parent_ids.each_with_index
```

Comment:
Are we sure DHIS2 provides the parents in a correct (non bulk) order?

Reply:
yes path is a string like this one:  /country_id/region_id/.../fosa_id